openapi/composer/openapi.yml

```diff
@@ -0,0 +1,230 @@
+          items:
+            $ref: '#/components/schemas/Repository'
+        upload_requests:
+          type: array
+          items:
+            $ref: '#/components/schemas/UploadRequest'
+    Repository:
+      type: object
+      required:
+        - baseurl
+      properties:
+        baseurl:
+          type: string
+          format: url
+          example: 'https://cdn.redhat.com/content/dist/rhel8/8/x86_64/baseos/os/'
+    UploadRequest:
+      oneOf:
+        - $ref: '#/components/schemas/AWSUploadRequest'
+    AWSUploadRequest:
+      type: object
+      required:
+        - type
+        - options
+      properties:
+        type:
+          type: string
+          enum: ['aws']
+        options:
+          type: object
+          required:
+            - region
+            - s3
+            - ec2
+          properties:
+            region:
+              type: string
+              example: 'eu-west-1'
+            s3:
+              type: object
+              required:
+                - access_key_id
+                - secret_access_key
+                - bucket
+              properties:
+                access_key_id:
+                  type: string
+                  example: 'AKIAIOSFODNN7EXAMPLE'
+                secret_access_key:
+                  type: string
+                  format: password
+                  example: 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+                bucket:
+                  type: string
+                  example: 'my-bucket'
+            ec2:
+              type: object
+              required:
+                - access_key_id
+                - secret_access_key
+              properties:
+                access_key_id:
+                  type: string
+                  example: 'AKIAIOSFODNN7EXAMPLE'
+                secret_access_key:
+                  type: string
+                  format: password
+                  example: 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+    Customizations:
+      type: object
+      properties:
+        subscription:
+          type: object
+          required:
+            - organization
+            - activation-key
+            - server-url
+            - base-url
+            - insights
+          properties:
+            organization:
+              type: integer
+              example: 2040324
+            activation-key:
+              type: string
+              format: password
+              example: 'my-secret-key'
+            server-url:
+              type: string
+              example: 'subscription.rhsm.redhat.com'
+            base-url:
+              type: string
+              format: url
+              example: http://cdn.redhat.com/
+            insights:
+              type: boolean
+              example: true
+    ComposeResponse:
+      required:
+        - compose_id
+      properties:
+        compose_id:
+          type: string
+          format: uuid
+          example: '123e4567-e89b-12d3-a456-426655440000'
```

Comment:
Did you write this by hand? It lacks a newline at the end of the file. Same for the other yml.

Reply:
Sort of. Using the swagger.io editor. Fixed now.